vhdbuilder/packer/vhd-scanning.sh

```diff
@@ -74,7 +72,10 @@ az vm create --resource-group $RESOURCE_GROUP_NAME \
     --admin-password $SCAN_VM_ADMIN_PASSWORD \
     --os-disk-size-gb 50 \
     ${VM_OPTIONS} \
-    --assign-identity "${SCANNING_MSI_RESOURCE_ID}"
+    --assign-identity "[system]"
+
+VM_PRINCIPLE_ID=$(az vm identity show --name $VM_NAME --resource-group $RESOURCE_GROUP_NAME --query principalId --output tsv)
```

Comment:
we need to be setting VM_NAME

Reply:
I might have tweeked something the wrong way with a merge conflict

Reply:
jk looks like it should just be $SCAN_VM_NAME instead of $VM_NAME